DigitalLearningSolutions.Data/DataServices/GroupsDataService.cs

```diff
@@ -129,6 +139,33 @@ FROM CentreApplications
                             AND GDInner.DelegateID = P.CandidateID
                             AND GCInner.GroupID != GC.GroupID)";
 
+        private readonly string GroupsSql = @$"SELECT

```

Comment:
I tried making it a const but it doesn't let me because it has the CourseCountSql interpolated within it. And yes I've applied it to the GetGroupsForCentre method